### PR TITLE
Intake to Main

### LINKED ISSUE #8: Intake subsystem
Create one intake subsystem class to represent both intakes on the bot. the constructor should take in the CAN ID of the motor controller and the port numbers on the PCM as parameters.

Methods
- intakeSet takes in boolean True if extended False is retracted
- intakeGet returns boolean True if extended False is retracted
- intakeSpeedSet takes in a double which represents percentage speed to set the intake motor to
- intakeSpeedGet returns double which represents percentage speed to set the intake motor to

### PR DESCRIPTION
**Pull request by Spenceroni-Pepperoni**
Closes #8

### CODE REVIEW

File: simgui-ds.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
```
Comment (KevinH45) [inline simgui-ds.json:1]: What is this for?

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+//power of the motor when active
+double motorPower = 0.0;
+//take double parameter as the percentage of power given to each motor
+DoubleSolenoid solenoid1 = new DoubleSolenoid(PneumaticsModuleType.REVPH, 0, 1);
+DoubleSolenoid solenoid2 = new DoubleSolenoid(PneumaticsModuleType.REVPH, 2, 3);
+public boolean intakeGet() {
+    return (solenoid1.get().equals(DoubleSolenoid.Value.kForward) && solenoid2.get().equals(DoubleSolenoid.Value.kForward));
+}
+public void intakeSet(boolean pistonState) { //up false, down true
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java:29]: Fix indentation and re-add documentation.

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+ * This subsystem represents the intake system on the robot, it will set the
+ * motor speed in percent and set pistons to on or off
+ */
+public class IntakeSubsystem extends SubsystemBase {
+//PS: SillyPandaDog is also Jokorie
+//Initialize Intake Motors
+TalonSRX Intake = new TalonSRX(Constants.MOTOR_ID);
+
+//power of the motor when active
+double motorPower = 0.0;
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java:22]: Remove unnecessary variable.

**Review by liam-a-wms — CHANGES_REQUESTED** [review]

**Review by KevinH45 — APPROVED** [review]
Everything made sense when J3 explained it to me. Looks good.

**Review by Gliu1211 — APPROVED** [review]
Looks good

**Pull request merged**